operator-common/src/main/java/io/strimzi/operator/cluster/model/ClientsCa.java

```diff
@@ -37,6 +40,20 @@ public static Secret adapt060ClientsCaSecret(Secret clientsCaKey) {
         return clientsCaKey;
     }
 
+    public void initCaSecrets(Secret secret) {
+        this.brokersSecret = secret;
+    }
+
+    @Override
+    protected String caCertThumbprintAnnotation() {
+        return "clients-ca-thumbprint";
```

Comment:
Same as earlier. I think this would be good in `Annotations` to have all annos centrally. (not necessarily the method => just the string definition)

Reply:
Yeah, ditto as above.